examples/helloworld/send_text_to_room.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+from markdown import markdown
+
+from mautrix.client import ClientAPI
+
+user_id = "@admin:example.com"
+base_url = "https://example.com"
+token = "syt_123_456"
+
+client = ClientAPI(user_id, base_url=base_url, token=token)
+
+
+async def send_message_to_room(
+    room_id: str, message: str, notice: bool = False, markdown_convert: bool = False
+) -> None:
+
+    msgtype = "m.notice" if notice else "m.text"
+
+    content = {
+        "formatted_body": None,
+        "body": message,
+        "msgtype": msgtype,
+    }
+
+    if markdown_convert:
+        content["formatted_body"] = markdown(message)
```

Comment:
I think we should encourage the use of the nicer `TextMessageEventContent` object in this situation. Also, need to specify format.

```suggestion
    content = TextMessageEventContent(
        msgtype=msgtype,
        body=message,
        format=Format.HTML,
        formatted_body=markdown(message),
    )
```

Reply:
I will take your advice, thank you very much :3 @sumnerevans